client/my-sites/checkout/composite-checkout/hooks/use-site-domains.ts

```diff
@@ -12,7 +12,9 @@ export default function useSiteDomains( siteId: number | undefined ): SiteDomain
 
 	const [ siteDomains, setSiteDomains ] = useState< SiteDomain[] >( [] );
 
-	const areDomainsLoaded = useSelector( ( state ) => hasLoadedSiteDomains( state, siteId ) );
+	const areDomainsLoaded = useSelector( ( state ) =>
+		hasLoadedSiteDomains( state, siteId ?? null )
+	);
```

Comment:
if `siteId` is `null` or `0` we can probably skip asking about it

```ts
useSelector( ( state ) => siteId ? hasLoadedSiteDomains( state, siteId ) : false );
```

Reply:
You're right. I only did this in places where the selector already accepted a falsy value, but in places like this, where it's obvious that we want a boolean, we can skip the selector entirely.